swagger2.0/oic.wk.res.swagger.json

```diff
@@ -43,116 +97,52 @@
-        "parameters": [
-          {"$ref": "#/parameters/interface-baseline"}
-        ],
-        "responses": {
-            "200": {
-              "description" : "",
-              "x-example":
-                [
-                  {
-                  "rt": ["oic.wk.res"],
-                  "if": ["oic.if.baseline", "oic.if.ll" ],
-                  "links":
-                    [
-                      {
-                        "href": "/humidity",
-                        "rt":   ["oic.r.humidity"],
-                        "if":   ["oic.if.s"],
-                        "p":    {"bm": 3},
-                        "eps": [
-                            {"ep": "coaps://[fe80::b1d6]:1111", "pri": 2},
-                            {"ep": "coaps://[fe80::b1d6]:1122"},
-                            {"ep": "coap+tcp://[2001:db8:a::123]:2222", "pri": 3}
-                        ]
-                      },
-                      {
-                        "href": "/temperature",
-                        "rt":   ["oic.r.temperature"],
-                        "if":   ["oic.if.s"],
-                        "p":    {"bm": 3},
-                        "eps": [
-                            {"ep": "coaps://[[2001:db8:a::123]:2222"}
-                        ]
-                      }
-                    ]
-                  }
-                ],
-              "schema": { "$ref": "#/definitions/sbaseline" }
+                ]
+              }
+            ],
+            "schema": {
+              "$ref": "#/definitions/sbaseline"
             }
+          }
         }
       }
     }
   },
   "parameters": {
-    "interface-ll" : {
-      "in" : "query",
-      "name" : "if",
-      "type" : "string",
-      "enum" : ["oic.if.ll"]
+    "interface-ll": {
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.ll"]
     },
-    "interface-baseline" : {
-      "in" : "query",
-      "name" : "if",
-      "type" : "string",
-      "enum" : ["oic.if.baseline"]
+    "interface-baseline": {
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.baseline"]
     },
-    "interface-all" : {
-      "in" : "query",
-      "name" : "if",
-      "type" : "string",
-      "enum" : ["oic.if.ll", "oic.if.baseline"]
+    "interface-all": {
+      "in": "query",
+      "name": "if",
+      "type": "string",
+      "enum": ["oic.if.ll", "oic.if.baseline"]
     }
   },
   "definitions": {
-    "slinklist" : {
-      "items" : {
+    "slinklist": {
+      "items": {
         "properties": {
           "anchor": {
-            "description": "This is used to override the context URI e.g. override the URI of the containing collection.",
-            "format": "uri",
-            "maxLength": 256,
-            "type": "string"
+            "$ref": "https://openconnectivityfoundation.github.io/core/schemas/oic.links.properties-schema.json#/definitions/anchor"
           },
           "di": {
```

Comment:
This seems like the old /oic/res response that included "di".  I'm not sure why this is still here...